src/components/case_manager/MyParas.tsx

```diff
@@ -4,24 +4,38 @@ import styles from "../../styles/Dashboard.module.css";
 import Link from "next/link";
 import PersonCreationForm from "./PersonCreationForm";
 
-const AllParasPage = () => {
+const MyParas = () => {
   const utils = trpc.useContext();
-  const { data: paras, isLoading } = trpc.para.getAllParas.useQuery();
-  const { mutate } = trpc.para.createPara.useMutation({
-    onSuccess: () => {
-      return utils.para.getAllParas.invalidate();
+  const { data: paras, isLoading } = trpc.para.getMyParas.useQuery();
+
+  const createPara = trpc.para.createPara.useMutation({
+    onSuccess: () => utils.para.getMyParas.invalidate(),
+    onSettled: (data, error) => {
+      if (error) {
+        return alert(error.message);
```

Comment:
🐧 maybe log to console instead for now? hopefully https://github.com/sfbrigade/compass/issues/84 is in soon